### PR TITLE
Porting GraphExplorer proj to .NETCore

### PR DESCRIPTION
**Pull request by sstchur**
Created a new folder, as a sibling of Web, called Web.NetCore, which contains a new project build on .NETCore with code ported form Web/GraphExplorer

Instructions to build:
From the Web.NetCore directory:
npm run fullbuild
dotnet restore
dotnet run

Notes: hot module reload is supported but you need to make sure your ASPNETCORE_Environment is set to "Development"

Known issues:
Not currently loading in IE11.  Bluebird polyfill is in and loading, but Aurelia is bombing out somewhere within routing (stack overflow or something).

### CODE REVIEW

File: Web.NetCore/GraphExplorer/App_Data/graphsettings.json [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+      "edgeSize": 1,
+      "iconSize": 25,
+      "defaultNodeShape": "box",
+      "labelMappings": {},
+      "physicsSolver": "forceAtlas2Based",
+      "edgeColors": {}
+    },
+    "showEdgeLabel": false
+  },
+  "build2017-sample-data": {
```
Comment (jasmcn) [inline Web.NetCore/GraphExplorer/App_Data/graphsettings.json]: You'll probably want to remove the extra settings from this file.

File: Web.NetCore/GraphExplorer/App_Data/queries.json [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+      "title": "Get subgraph of a specific enterprise subscription",
+      "query": "g.V().union(has('EnterpriseSubscriptionId', 'O:8E44A22E-A985-44F8-BF8D-13BDAEA5DF89'),has('EnterpriseSubscriptionId', 'O:8E44A22E-A985-44F8-BF8D-13BDAEA5DF89').out(),has('EnterpriseSubscriptionId', 'O:8E44A22E-A985-44F8-BF8D-13BDAEA5DF89').in(),has('TenantId','A:0034E878-B23B-4468-B7E8-9597F3CCBDD1').out(),has('TenantId','A:0034E878-B23B-4468-B7E8-9597F3CCBDD1').in(), has('TenantId','A:00283E9B-FEE2-4E69-8E80-A85C79F306C5').out(),has('TenantId','A:00283E9B-FEE2-4E69-8E80-A85C79F306C5').in()); g.E();"
+    },
+    {
+      "id": "15",
+      "title": "Get all Subscriptions with more than one Tenant",
+      "query": "g.V().union(__.where(in('TenantEnterpriseSubscription').count().is(gt(1))),__.where(in('TenantEnterpriseSubscription').count().is(gt(1))).repeat(__.in()).times(2).emit(), __.where(in('TenantEnterpriseSubscription').count().is(gt(1))).out()); g.E();"
+    }
+  ],
+  "build2017-sample-data": [
```
Comment (jasmcn) [inline Web.NetCore/GraphExplorer/App_Data/queries.json]: Same here.

File: Web.NetCore/GraphExplorer/appsettings.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "DocumentDBConfig":
+  {
+    "endpoint": "https://build2017-official.documents.azure.com:443/",
+    "authKey": "aw2hJzSwVKsrtC2RKITD5JIzYAKaKnsedRLMuAmsmg7yemyf9q4w4GTLKwX9H3U9Kw9MDSBS19LP1lHwRS0a8w==",
```
Comment (sstchur) [inline Web.NetCore/GraphExplorer/appsettings.json]: This should not be included in the PR

Comment (jasmcn) [inline Web.NetCore/GraphExplorer/appsettings.json]: Welp, it's public now; time to regenerate the authkey!

File: Web.NetCore/GraphExplorer/Configuration/DocDbSetup.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿
+using System;
+
+namespace GraphExplorer.Configuration
+{
+    // public static class DocDbSettings
```
Comment (jasmcn) [inline Web.NetCore/GraphExplorer/Configuration/DocDbSetup.cs]: Why's this commented out?

File: Web.NetCore/GraphExplorer/appsettings.json (hunk 3 above)
Comment (sstchur) [inline Web.NetCore/GraphExplorer/appsettings.json]: I did already

**Pull request merged**